tb_plugin/torch_tb_profiler/profiler/loader.py

```diff
@@ -54,17 +55,31 @@ def _process(self):
         for data in self.run.profiles.values():
             logger.debug("Processing profile data")
             data.process()
-            comm_node_lists.append(data.comm_node_list)
+            if not data.has_communication:
```

Comment:
so, we would turn the distributed feature off in case of any workers doesn't have the distributed data. right? 

Suggest to add comments here about the reason